### PR TITLE
[TS-74] Add loading state to tables in profile page 

### PR DESCRIPTION
**Pull request by William-Unosquare**
When data is being fetched the tables will now display as loading.

### CODE REVIEW

File: web-app/src/components/DataTable/index.jsx [hunk 1]
```diff
@@ -59,15 +62,17 @@ class DataTable extends Component {
...
     const formattedColumns = this.buildColumns(columns);
 
     const labels = formattedColumns.reduce((acc, column) => {
       acc.push(column.Header);
       return acc;
     }, []);
 
-    if (data.length > 0) {
+    if (data.length < 0 && !loading) {
+      return <p>{emptyMessage}</p>;
```
Comment (ghost) [inline web-app/src/components/DataTable/index.jsx:74]: emptyMessage?

Comment (William-Unosquare) [reply web-app/src/components/DataTable/index.jsx:74]: It’s the message that’s displayed when there’s no data. May get rid of it honestly and just show an empty table instead.

Comment (ghost) [reply web-app/src/components/DataTable/index.jsx:74]: Ah I see, yeah whatever you think is better. For a second I was wondering whether it was just empty that variable haha.

**Review by ghost — APPROVED** [review]

File: web-app/src/components/DataTable/index.jsx (hunk 1 above)
Comment (William-Unosquare) [reply web-app/src/components/DataTable/index.jsx:74]: I'll keep it like this for now just so I can work on more MVP critical issues.

**Pull request merged**